include/highfive/bits/H5Attribute_misc.hpp

```diff
@@ -27,6 +27,24 @@
 
 namespace HighFive {
 
+inline std::string Attribute::getAttributeName() const {
+    const size_t maxLength = 255;
+    char  buffer[maxLength + 1];
+    ssize_t retcode = H5Aget_name(
```

Comment:
Seems to me that this code repeats itself several times, save for this little function. Could probably be deduplicated? Also the naming could be shortened to `getName` and `getPath` without repeating the class name in the method name…

Reply:
Hi,
I absolutely agree with your proposition about changing the names of the methods. So now they are called `getName` and `getPath`
> Seems to me that this code repeats itself several times, save for this little function. Could probably be deduplicated?

I still don't fully understand how this library works because I have huge number of errors inside the project :)
I guess this is beacause one namespace is divided in many `.hpp` files and when I compile the code it works because those files are combined I think. 
But editing the code is difficult in such conditions. In this case creating new small functions that decrease the number of code lines would make the code less readable for me and probably for other developpers who probably would read this code.
So if you don't mind I would leave it as is.